aspnetcore/blazor/hosting-models.md

```diff
@@ -35,24 +37,26 @@ The Blazor WebAssembly hosting model offers several benefits:
 * There's no .NET server-side dependency. The app is fully functioning after it's downloaded to the client.
 * Client resources and capabilities are fully leveraged.
 * Work is offloaded from the server to the client.
-* An ASP.NET Core web server isn't required to host the app. Serverless deployment scenarios are possible (for example, serving the app from a CDN).
+* An ASP.NET Core web server isn't required to host the app. Serverless deployment scenarios are possible, such as serving the app from a Content Delivery Network (CDN).
 
-There are downsides to Blazor WebAssembly hosting:
+There are downsides to using the Blazor WebAssembly hosting model:
```

Comment:
This sounds too negative, and comes along as `don't use this model, as it's bad`.

```suggestion
Blazor WebAssembly hosting model has some limitations, which you have to be aware of:
```